test/integration/functional_test.go

```diff
@@ -143,6 +145,28 @@ func TestFunctional(t *testing.T) {
 			})
 		}
 	})
+
+}
+
+func cleanupUnwantedImages(ctx context.Context, t *testing.T, profile string) {
+	if ContainerRuntime() == "docker" {
```

Comment:
@BLasan this check should change to if "Docker" is installed on the user machine

Reply:
Yuh, will fix this.